doc/docs/api/env.md

```diff
@@ -0,0 +1,106 @@
+# Environmental variables
```

Comment:
Maintaining this separate from the code is very error prone. We very likely want to generate it from the code itself.